controllers/util/tests/tests.go

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright 2019 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tests
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	csiv1 "github.com/IBM/ibm-block-csi-operator/api/v1"
+	"github.com/IBM/ibm-block-csi-operator/pkg/config"
+	"gopkg.in/yaml.v2"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var (
+	relativeCrPath = "../config/samples/csi.ibm.com_v1_ibmblockcsi_cr.yaml"
+	nodeContainerName = "ibm-block-csi-node"
+	controllerContainerName = "ibm-block-csi-controller"
```

Comment:
could these [be](https://github.com/IBM/ibm-block-csi-operator/blob/develop/controllers/syncer/csi_node.go#L40) [imported](https://github.com/IBM/ibm-block-csi-operator/blob/develop/controllers/syncer/csi_controller.go#L41)?

Reply:
done